src/schema.js

```diff
@@ -199,14 +199,20 @@ const createGifSchema = Joi.object({
 		published: Joi.boolean()
 	}
 })
+const getGifByIdSchema = Joi.object({
```

Comment:
@beblicarl i personally don't see any reason we can't use the `getPostByIdSchema` here

they're basically the same

Reply:
yes please.